src/config/locales/fr/translation.json

```diff
@@ -121,7 +121,10 @@
       "cw": "Écrivez votre avertissement ici",
       "status": "Qu'avez-vous en tête ?",
       "cancel": "Annuler",
-      "toot": "Pouet!"
+      "toot": "Pouet!",
+      "close_confirm": "Êtes-vous sûr de vouloir fermer?",
+      "close_confirm_ok": "Redémarrer",
+      "close_confirm_cancel": "Annuler"
```

Comment:
@gfrenoy Is this correct? I don't know French.

Reply:
"close_confirm": "Êtes-vous certain de vouloir fermer ce nouveau pouet ?",
"close_confirm_ok": "Oui",

Reply:
Thank you! :bowing_man: